client/account-recovery/forgot-username/forgot-username-form/index.jsx

```diff
@@ -13,79 +14,132 @@ import Button from 'components/button';
+			return translate( "We're not able to find an account matching that information. " +
+				'Double-check your spelling, or try another name or URL.' );
+		}
+
+		return translate( "We've encountered some technical issues. Please try again later." );
+	};
+
+	render() {
+		const {
+			translate,
+			userData,
+			isRequesting,
+			requestError,
+		} = this.props;
+
+		const {
+			firstName,
+			lastName,
+			url,
+		} = userData;
+
+		const isPrimaryButtonEnabled = firstName && lastName && url && ! isRequesting;
 
 		return (
-			<div>
+			<Card>
 				<h2 className="forgot-username-form__title">
 					{ translate( 'Forgot your username?' ) }
 				</h2>
-				<p>{ translate( 'Enter your information to find your username' ) }</p>
-				<Card>
-					<FormLabel>
-						{ translate( 'First Name' ) }
-						<FormInput
-							className="forgot-username-form__first-name-input"
-							onChange={ this.firstNameUpdated }
-							value={ firstName }
-							disabled={ isSubmitting } />
-					</FormLabel>
-					<FormLabel>
-						{ translate( 'Last Name' ) }
-						<FormInput
-							className="forgot-username-form__last-name-input"
-							onChange={ this.lastNameUpdated }
-							value={ lastName }
-							disabled={ isSubmitting } />
-					</FormLabel>
-					<FormLabel>
-						{ translate( "Your site's URL" ) }
-						<FormInput
-							className="forgot-username-form__site-url-input"
-							onChange={ this.siteUrlUpdated }
-							value={ siteUrl }
-							disabled={ isSubmitting } />
-					</FormLabel>
-					<Button
-						className="forgot-username-form__submit-button"
-						onClick={ this.submitForm }
-						disabled={ ! isPrimaryButtonEnabled }
-						primary>
-						{ translate( 'Continue' ) }
-					</Button>
-				</Card>
-			</div>
+				<p>{ translate( 'Enter your full name and URL instead.' ) }</p>
+				<FormLabel>
+					{ translate( 'First Name' ) }
+					<FormInput
+						className="forgot-username-form__first-name-input"
+						onChange={ this.firstNameUpdated }
+						value={ firstName || '' }
+						disabled={ isRequesting } />
+				</FormLabel>
+				<FormLabel>
+					{ translate( 'Last Name' ) }
+					<FormInput
+						className="forgot-username-form__last-name-input"
+						onChange={ this.lastNameUpdated }
+						value={ lastName || '' }
+						disabled={ isRequesting } />
+				</FormLabel>
+				<FormLabel>
+					{ translate( "Your site's URL" ) }
+					<FormInput
+						className="forgot-username-form__site-url-input"
+						onChange={ this.siteUrlUpdated }
+						value={ url || '' }
+						disabled={ isRequesting } />
+				</FormLabel>
+				{
+					requestError && (
+					<p className="forgot-username-form__error-message">
+						{ this.getErrorMessage( requestError ) }
+					</p> )
+				}
+				<Button
+					className="forgot-username-form__submit-button"
+					onClick={ this.submitForm }
+					disabled={ ! isPrimaryButtonEnabled }
+					primary>
+					{ translate( 'Continue', { context: 'verb' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Continue' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/account-recovery/state-plumbing-for-route-forgot-username)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.